test-float-features.lisp

```diff
@@ -0,0 +1,107 @@
+(in-package #:cl-user)
+
+(defpackage #:float-features/tests
+  (:nicknames #:org.shirakumo.float-features/tests)
+  (:use #:cl))
+
+;;; to run (parachute:test :float-features/tests)
+(in-package #:float-features/tests)
+
+(parachute:define-test float-features-1
+    :compile-at :execute
+    (parachute:true
+     (float-features:float-infinity-p
+      (flet ((foo () (if (> 10 (random 20)) 0.0 0.0))
+             (bar () (if (> 10 (random 20)) 23 24)))
+        (float-features:with-float-traps-masked (:divide-by-zero)
+          (/ (bar) (foo)))))))
+
+(parachute:define-test float-features-2
+     :compile-at :execute
+    (parachute:true
+     (handler-case
+         (flet ((foo () (if (> 10 (random 20)) 0.0 0.0))
+                (bar () (if (> 10 (random 20)) 23 24)))
+           (float-features:with-float-traps-masked ()
+             (/ (bar) (foo))
+             ))
```

Comment:
Do not leave close parens on their own lines.

Reply:
done